Firebase/Auth/Source/AuthProviders/Phone/FIRPhoneAuthProvider.m

```diff
@@ -38,6 +38,8 @@
 #import "FIRSendVerificationCodeResponse.h"
 #import "FIRVerifyClientRequest.h"
 #import "FIRVerifyClientResponse.h"
+#import <GoogleToolboxForMac/GTMNSDictionary+URLArguments.h>
+#import <GTMSessionFetcher/GTMSessionFetcherService.h>
```

Comment:
I don't see GTMSessionFetcher is used here?

Reply:
Was used on line 391 to get part of the user agent string.
`@"v" : [NSString stringWithFormat:@"FirebaseAuth.iOS/%s %@",
                                        FirebaseAuthVersionString,
                                        GTMFetcherStandardUserAgentString(nil)]`

Replaced with  the class method on backend now, so removed.